src/DependencyInjection/Configuration.php

```diff
@@ -28,8 +28,14 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode    = $treeBuilder->root('sonata_helpers');
+        $treeBuilder = new TreeBuilder('sonata_helpers');
```

Comment:
should be in the if/else block because no explicit constructor for Symfony < 4.2.0